swag/db.py

```diff
@@ -197,6 +260,63 @@ def creation_date(self, value):
         self.swagdb.creation_date[self.id] = value
 
 
+class Cagnotte:
+    def __init__(self, swagdb, cagnotte_id) -> None:
+        self.swagdb = swagdb
+        self.id = cagnotte_id
+
+    def get_info(self):
+        return CagnotteInfo(self)
+
+    @property
+    def name(self):
+        return self.swagdb.cagnotte_name[self.id]
+
+    @name.setter
+    def name(self, value):
+        self.swagdb.cagnotte_name[self.id] = value
+
+    @property
+    def balance(self):
+        return self.swagdb.cagnotte_balance[self.id]
+
+    @balance.setter
+    def balance(self, value):
+        self.swagdb.cagnotte_balance[self.id] = value
+
+    @property
+    def currency(self):
+        return self.swagdb.cagnotte_currency[self.id]
+
+    @currency.setter
+    def currency(self, value):
+        self.swagdb.cagnotte_currency[self.id] = value
+
+    @property
+    def manager(self):
+        return self.swagdb.cagnotte_manager[self.id]
+
+    @manager.setter
+    def manager(self, value):
+        self.swagdb.cagnotte_manager[self.id] = value
+
+    @property
+    def participant(self):
```

Comment:
`participants`

Reply:
Resolved in : https://github.com/BobbySoftwares/Bybbot/pull/7/commits/c9cfecdb424780b7243e2b2ba8fcc0fba8a01e8f